### ISSUE TITLE
Dropout have not been shutdown in test mode in RGCN

### ISSUE THREAD

**Issue by Lee-zix** [dmlc/dgl#731]
Great project! It seems that dropout have not been shut down in RGCN when test, it seems that the codes do not distinguish the train mode and test mode in RGCN, Did i have some misunderstandings about the RGCN codes?? Thanks for any help!!!!

**Comment by VoVAllen** [comment]
Code in tutorial is simplified for readiness. You can find the whole code [here](https://github.com/dmlc/dgl/tree/master/examples/pytorch/rgcn), which includes the train and test mode.

**Comment by laifi** [comment]
This is what [model.eval()](https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link_predict.py#L183) supposed to do

**Comment by yzh119** [comment]
@laifi yes, and we indeed called `model.eval()` in our examples:
https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/entity_classify.py#L121
https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link_predict.py#L207

**Comment by Lee-zix** [comment]
Yeah, i got it. Thanks for your timely reply
